lib/Span/SpanProcessor.php

```diff
@@ -151,6 +156,11 @@ private function replaceLinks(string $span) : string
             /** @var string $link */
             $link = $match[3] ?: $match[5];
 
+            // a link starting with _ is not a link - return original string
+            if (substr($link, 0, 1) === '_') {
+                return $match[0];
+            }
```

Comment:
This callback is also called for normal links, and so a weird link like this:

```
A link starting with an underscore: `_CLASS`_
```

WILL be affected by this change and will no longer be seen as a valid link. In Sphinx, this IS seen as a valid link, but I couldn't find any valid syntax at the bottom of the page to actually supply this URL - e.g. this doesn't work:

```rst
.. _`_CLASS`: https://symfony.com
```

So, I think this might not even be a supported case. We can't make this function ONLY affect anonymous links (I tried that) because after calling this callback for anonymous links, it is then called for normal links. That was causing the `__CLASS__` to THEN be seen incorrect as a normal link (e.g. for `__CLASS_`) that was broken.